src/mixins/filing-template-mixin.ts

```diff
@@ -524,7 +524,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
 
     // store Business Information for BEN corrections
     if (this.isBenCorrectionFiling) {
-      this.setBusinessInformation({ ...filing.business })
+      this.setBusinessInformation({
+        ...entitySnapshot.businessInfo,
+        ...filing.business
+      })
```

Comment:
The old code overwrote all the good business info (like start date) from whatever was in the draft filing. Oops.